src/App/Fossa/Project/Edit.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module App.Fossa.Project.Edit (
+  editMain,
+) where
+
+import App.Fossa.Config.Project.Edit (EditConfig (..))
+import Control.Algebra (Has)
+import Control.Carrier.Diagnostics (context)
+import Control.Effect.Diagnostics (Diagnostics, warn)
+import Control.Effect.FossaApiClient (FossaApiClient, editProject, getOrganization)
+import Control.Effect.Lift (Lift)
+import Control.Monad (when)
+import Data.Foldable (traverse_)
+import Effect.Logger (Logger, logInfo, logStdout)
+import Fossa.API.Types (Organization (..), ProjectResponse (..))
+
+editMain ::
+  ( Has Diagnostics sig m
+  , Has Logger sig m
+  , Has (Lift IO) sig m
+  , Has FossaApiClient sig m
+  ) =>
+  EditConfig ->
+  m ()
+editMain EditConfig{..} = do
+  logInfo "Running FOSSA project edit"
+  org <- getOrganization
+  when (orgSupportsProjects org) $ do
```

Comment:
I think we should give some sort of explanation if the org does not suport projects. Right now it just fails silently. For example, here's what it looks like when I run it against production:

```
fossa-dev project edit --fossa-api-key $FOSSA_FULL_API_KEY --project-id custom+1/git@github.com/example --title newTitle --project-label abc --project-label xxx --debug
[DEBUG] Loading configuration file from "/Users/scott/fossa/fossa-cli/"
Running FOSSA project
Running FOSSA project edit
```